### DISCUSSION TITLE
Python Tip - Interpreter - not in History

### DISCUSSION THREAD

**Discussion by WishStone-Dragon** [Monika-After-Story/MonikaModDev#8557] [QA - tech support + general] [answered]
Hi team!

Unsure if this is wanted or not, but I just noticed that a bulk of the text from the 'interpreter' introduction isn't showing in the history when talking to Monika.

![interpreter](https://user-images.githubusercontent.com/95390551/147960285-4deb4937-b906-4067-8a49-404a9eee1955.PNG)

In the code, I can see this:

`    show monika 3eua at t22
    show screen mas_py_console_teaching

    m 3eub "this!"

    m "You can enter Python code directly into here and run it, like so:"
    show monika 3eua

    # base commands shown as starter ones
    call mas_wx_cmd("12 + 3", local_ctx)
    call mas_wx_cmd("7 * 6", local_ctx)
    call mas_wx_cmd("121 / 11", local_ctx)
    # NOTE: add more commands as the user goes thru the tips

    if mas_getEVL_last_seen("monika_ptod_tip003") is None:
        m 1eua "You can do more than just math using this tool, but I'll show you all of that as we go along."

        m 1hksdlb "Unfortunately, since this is a fully functional Python interpreter and I don't want to risk you accidentally deleting me or breaking the game,"
        m "Not that you would{fast}{nw}"
        $ _history_list.pop()
        m 1eksdlb "I can't let you use this.{w=0.3} Sorry..."
        m "If you want to follow along in future lessons, then run a Python interpreter in a separate window."

        m 1eua "Anyway, I'll be using {i}this{/i} interpreter to help with teaching."

    else:
        m 1hua "Pretty cool, right?"

    $ store.mas_ptod.ex_cn()
    hide screen mas_py_console_teaching
    show monika at t11

    m 1hua "Thanks for listening!"`

Is this happening due to **m "Not that you would{fast}{nw}"**? I ask because I noticed that {nw} lines, in general, don't show in the history.

Is this wanted behaviour in the history?

**Comment by multimokia** [comment, marked as answer]
This is intended

**Reply by WishStone-Dragon** [reply]
Roger, thank you!
